shared/fixes/bash/rsyslog_files_permissions.sh

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel
+# platform = multi_platform_rhel,Oracle Linux 7
```

Comment:
You're kinda inconsistent on `multi_platform_ol` versus `Oracle Linux 7`; is there a particular reason why this needs to be OL7-only?

Reply:
At the moment only OL7 support added. While some profiles in the future might be OL6 specific and due to some differences between releases like grub/grub2 multi_platform won't work here.
But in this case there is no reason to use specific platform name. I'll try to be more consistent in the future :)

Reply:
Ahh okie, then either way is fine by me. :)